xbmc/test/TestUtils.cpp

```diff
@@ -7,6 +7,8 @@
  */
 
 #include "TestUtils.h"
+
+#include "FileItem.h"
```

Comment:
Please remove. Not needed.

Reply:
It doesn't compile without.

Reply:
Compiler error output, please.

Reply:
Severity	Code	Description	Project	File	Line	Suppression State
Error	C2143	syntax error: missing ',' before '&' (compiling source file C:\Code\xbmc\xbmc\test\TestUtils.cpp)	kodi-test	C:\Code\xbmc\xbmc\Util.h	232

Reply:
This is not caused by "CFilUtils.cpp". Nees to be fixed in "Util.h". There, `#include "FileItem.h` is missing. You added there a method having a `CFileItemPtr` as parameter. For that, FileItem.h needs to be included.